userspace/libscap/engine/kmod/scap_kmod.c

```diff
@@ -74,17 +74,66 @@ static uint32_t get_max_consumers()
 	return 0;
 }
 
+/// TODO: we need to pass directly the system syscall number not the `ppm_sc` here.
```

Comment:
```suggestion
// TODO: we need to pass directly the system syscall number not the `ppm_sc` here.
```
Not sure I understood this comment :thinking: